fiasko_bro/code_validator.py

```diff
@@ -136,6 +155,7 @@ class CodeValidator:
         'max_pep8_line_length': 100,
         'max_number_of_lines': 200,
         'max_indentation_level': 4,
+        'max_num_of_py_files': 10,
```

Comment:
Such a small limit will prevent `fiasko_bro` from expecting legit Django projects (some of which have 10+ `.py` files at the very start and can have a lot of migrations).

The whole purpose of the filter is to determine whether virtual environment is in the repo. And the thing with virtual environment  is that it usually huge (my directory `venv` for `fiasko_bro` has more than 500 Python files). 

So let's make it 100. My intuition suggests that `fiasko_bro` has no problems analyzing this much files. You are welcome, however, to verify this intuition on your projects. I like to  count `.py` files with this command: `find . -type f | grep '.py$' | wc -l`.